plugins/mongo_db_plugin/mongo_db_plugin.cpp

```diff
@@ -1427,39 +1435,44 @@ void mongo_db_plugin_impl::init() {
          }
 
          try {
+            // Due to the vast amounts of data, we suggest:
+            //   1. enableSharding database (default to EOS)
+            //   2. shardCollection: blocks, action_traces, transaction_traces, especially action_traces
+            //   3. Use compound index with shard key (default to _id), to improve query performance.
+
```

Comment:
Are you saying you would like something added to the mongo_db_plugin for 1 and 2 OR that you merely suggest users should enableSharding on there database and collections?

Reply:
I suggest users should do 1 and 2 on production environment.

Reply:
Can you update the comment to make that clearer.